app/lab/manifest.webmanifest

```diff
@@ -0,0 +1,21 @@
+{
+    "short_name": "JupyterLite",
+    "name": "JupyterLite",
+    "description": "WASM powered Jupyter running in the browser",
+    "icons": [
+      {
+        "src": "../icon-120x120.png",
```

Comment:
Probably we could also use the yellow icon instead of the orange so it's easier to recognize?

https://github.com/jupyterlite/jupyterlite/blob/main/docs/_static/icon.svg